src/trustedtypes.js

```diff
@@ -100,14 +127,20 @@ export const TrustedTypes = (function() {
     }
 
     /**
-     * Name property getter.
-     * Required by the enforcer to work with both the polyfilled and native
-     * type.
+     * Returns the wrapped string value of the object.
+     * @return {string}
      */
-    static get name() {
-      return 'TrustedURL';
+    valueOf() {
+      return privates(this).value;
     }
   }
+  /**
+   * Name property.
+   * Required by the enforcer to work with both the polyfilled and native type.
+   */
+  defineProperty(
```

Comment:
Good catch! I thought `name` is just for error messages, but we actually rely on it in the polyfill enforcer.

Reply:
```js
> class C {}
< undefined
> Object.getOwnPropertyDescriptor(C, 'name')
< {value: "C", writable: false, enumerable: false, configurable: true}
```

so the type already has a static name property that is frozen.  We just need to make it unconfigurable.

Fixed.

Reply:
The compiler will usually rename the class.